app/assets/stylesheets/components/_status.scss

```diff
@@ -6,7 +6,12 @@
 }
 
 .status__icon--exclamation {
-  fill: $color-yellow-dark;
+  background-color: $color-yellow-light;
+  border-radius: 100%;
```

Comment:
Interesting that 100% worked, I'm sure this is usually meant to be set to 50% to create the circle.